test/testpy/__init__.py

```diff
@@ -29,7 +29,7 @@
 import os
 import re
 from functools import reduce
-from io import open
+import builtins
```

Comment:
Can we drop this?  Does it come from https://pypi.org/project/fissix, https://pypi.org/project/future, or https://pypi.org/project/modernize?  Is `builtins.open` just an alias to `open` in all supported versions of Python?